src/main/kotlin/com/workos/user_management/builders/AuthenticationWithMagicAuthOptionsBuilder.kt

```diff
@@ -0,0 +1,47 @@
+package com.workos.user_management.builders
+
+import com.workos.user_management.types.AuthenticationAdditionalOptions
+import com.workos.user_management.types.AuthenticationWithMagicAuthOptions
+
+/**
+ * Builder for options when authenticating with MFA.
```

Comment:
This should be "Magic Auth", since it is in `AuthenticationWithMagicAuthOptionsBuilder`, right?

```suggestion
 * Builder for options when authenticating with Magic Auth.
```

Reply:
👍